async/build.gradle

```diff
@@ -47,4 +47,8 @@ dependencies {
     implementation Dependencies.androidx_work_runtime
 
     testImplementation Dependencies.hamcrest
+    testImplementation Dependencies.robolectric
```

Comment:
This **should** work without Robolectric - the AndroidX Test stuff should be enough as you're not referencing Robolectric explicitly in the test. However, if I try and remove Robolectric, everything compiles but I get the following error:

```java
Caused by: java.lang.IllegalStateException: No instrumentation registered! Must run under a registering instrumentation.
	at androidx.test.platform.app.InstrumentationRegistry.getInstrumentation(InstrumentationRegistry.java:45)
	at androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner.createRunnerParams(AndroidJUnit4ClassRunner.java:53)
	at androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner.<init>(AndroidJUnit4ClassRunner.java:48)
```

I'm going to quickly dig into what's happening here as it's confusing.

Reply:
Ok yeah it's just that the docs are all wrong or everything is broken (standard): https://issuetracker.google.com/issues/219885372. Robolectric needs to be included to get AndroidX Test to work with "local tests".